ui/app/templates/vault/cluster.hbs

```diff
@@ -8,56 +8,52 @@
     <Nav.main>
       <ul class="navbar-sections {{if (has-feature 'Namespaces') 'with-ns-picker'}}">
         {{#if (has-feature "Namespaces")}}
-          <li>
-            <NamespacePicker @class="navbar-item" @namespace={{this.namespaceQueryParam}} />
-          </li>
+
+          <NamespacePicker @class="navbar-item" @namespace={{this.namespaceQueryParam}} />
+
         {{/if}}
-        <li class={{if (is-active-route "vault.cluster.secrets") "is-active"}}>
```

Comment:
I noticed this wasn't actually adding the `is-active` class to the tabs, so instead moved the logic to a class property on `<LinkTo>`